acoustic.ipynb

```diff
@@ -262,19 +266,23 @@
     "mask = ~parameter_table['Name'].isin(['R8', 'RR1', 'WITN', 'ENGI'])\n",
     "parameter_table = parameter_table[mask]\n",
     "\n",
+    "# Add units through the temporary units interface\n",
+    "units_column = []\n",
+    "for i, par in enumerate(f.parameters):\n",
+    "    if i in parameter_table.index:\n",
```

Comment:
This is just a matter of taste really, but I tend to flip the condition of if's and do a continue. The motivation is to have one less indentation level, which imo is easier to read:


```
for i, par in enumerate(f.parameters):
    if i not in parameter_table.index: continue
    try:
        ...
    except KeyError:
        ...
```

(Feel free to ignore this comment)